src/ResourceManager/MachineLearningCompute/Commands.MachineLearningCOmpute/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@shutchings there should be no `app.config` file for the `Commands` folder

Reply:
Got it. I think I had it because of some dependency issues but it looks like those got cleaned up in past revisions. Probably from the changes you made.